.github/workflows/performance-analysis.yml

```
@@ -34,7 +34,8 @@ jobs:
         sudo apt-get update
         sudo apt-get install build-essential git liblua5.1-dev zlib1g-dev libhunspell-dev libpcre3-dev \
           libzip-dev libboost-graph-dev libyajl-dev libpulse-dev lua-rex-pcre lua-filesystem lua-zip \
-          lua-sql-sqlite3 qtmultimedia5-dev qttools5-dev luarocks ccache libpugixml-dev cmake ninja-build -y
+          lua-sql-sqlite3 qtmultimedia5-dev qttools5-dev luarocks ccache libpugixml-dev cmake ninja-build
```

Don't you need a `\` at the end of this line to continue it into the next?
```suggestion
          lua-sql-sqlite3 qtmultimedia5-dev qttools5-dev luarocks ccache libpugixml-dev cmake ninja-build \
```